app/javascript/schools/courses/components/students_editor/StudentsEditor__StudentInfoForm.re

```diff
@@ -181,6 +191,29 @@ let make = (~addToListCB, ~studentTags, ~emailsToAdd) => {
         placeholder="Acme Inc., Acme University, etc."
       />
     </div>
+    <div className="mt-5">
+      <label
+        className="inline-block tracking-wide text-xs font-semibold mb-2 leading-tight"
+        htmlFor="team_name">
+        {"Team Name" |> str}
+      </label>
+      <span className="text-xs ml-1"> {"(optional)" |> str} </span>
+      <HelpIcon className="ml-1">
+        {"Students with same team name will be group together. This will now after existing students in the course"
```

Comment:
_Students with same team name will be grouped together; this will not affect existing teams in the course._